Components/Cards/cards.html

```diff
@@ -15,6 +15,7 @@
     <link rel="stylesheet" href="../Badge/badge.css" />
     <link rel="stylesheet" href="../../default.css" />
     <link rel="stylesheet" href="./cards.css" />
+    <link rel="stylesheet" href="../Rating/rating.css" />
   </head>
   <body class="flex-container">
     <div style="width: 100%; margin: 3rem;">
```

Comment:
avoid inline styling

Reply:
Oh. Thats temoporary styles. I will remove it, in documenattion.